### DISCUSSION TITLE
Access denied error when downloading some icons. Download link leads to an XML file.

### DISCUSSION THREAD

**Discussion by vikasbhatia** [elrumo/macOS_Big_Sur_icons_replacements#1494] [Help with the website]
When I try to download some icons, it goes to an XML file in the browser which says "The XML file does not appear to have any style information associated with it" and in the body of the document, it says access denied.

This happens randomly with some icons, not all. The domain it leads to is parsefiles.back4app.com

Here's a screen recording:

https://github.com/user-attachments/assets/dbb0ce75-9063-44c0-9a5e-b21141c5eee1

**Comment by pwiez** [comment]
I am having this exact same issue with some icons, including one for LibreOffice Calc. Did you ever find a solution for it or another way to download the icons?

**Reply by vikasbhatia** [reply]
Unfortunately no, sorry. If I do figure it out I'll post something here. You could try writing to the dev @elrumo or getting in touch via [Twitter](https://twitter.com/elrumo) to give them a nudge.

**Comment by IceBlueLugia** [comment]
Same issue on Google Meet

**Comment by exp-lab-usa** [comment]
When this happens, I just download the preview image from pop-up (right click > Save Image As…) then convert to .icns locally using Image2Icon.

When you submit icons on this website, you do so as PNG format at correct dimensions. That’s the file that shows as preview that you can download, so I’ve never had to adjust size or anything really. Extra steps, sure, but only way I found around this. Hopefully the developer fixes this as it has been an issue for quite some time now.

**Reply by vikasbhatia** [reply]
Good tip, thanks!

**Reply by milotek** [reply]
The problem with this, (for me at least) is that this is a low res version of the icon, which ticks me off because it's at a quarter size of the actual icon.

![CleanShot 2025-05-26 at 20 07 31@2x](https://github.com/user-attachments/assets/f1f0bc21-3907-4701-b8d6-8b38aa74e5a8)
![CleanShot 2025-05-26 at 20 08 06@2x](https://github.com/user-attachments/assets/96dc1b96-0fcb-4cda-87cd-5a4133e608b1)

It'd be nice if the maintainer could address this issue therefore!

**Reply by IceBlueLugia** [reply]
Honestly I’ve found it’s not really distinguishable for the most part. Even in Launchpad

**Reply by milotek** [reply]
Lucky you - I'm a perfectionist 😛

**Reply by exp-lab-usa** [reply]
Oh yeah that part. I actually only get that on some icons, not all. When I do, even on my 5k2k monitor, I rarely notice a difference. The few times I have, I just pass it through my upscale utility or Photoshop if I need to smooth/blend it. 

**To note, I also utilize a Chrome extension ([Imageye](https://chromewebstore.google.com/detail/agionbommeaifngbhincahgmoflcikhm?utm_source=item-share-cb)) (*Not affiliated and there are other options, just providing my utilized solution*) that finds all images on a page and allows me to download the highest resolution. That file might not be the one that is utilized for the thumbnail, which is why I don’t get those as often. 

Again, these are all last resort measures for regular users to get an icon they want without going to too much trouble. The real solution is the developer fixing the site. Fingers crossed as I utilize the site at least once every week and it remains the best repository of Mac icons I’ve found.
